src/main/java/kostka/moviecatalog/dto/RatingDto.java

```diff
@@ -1,19 +1,22 @@
 package kostka.moviecatalog.dto;
 
+import org.hibernate.validator.constraints.Range;
+
 import javax.validation.constraints.NotNull;
 
 public class RatingDto {
     @NotNull(message = "Id cannot be empty.")
-    private Long id;
+    private Long movieId;
     @NotNull(message = "Rating cannot be empty.")
+    @Range(min = 1, max = 10)
```

Comment:
Jde mi prave i o tento vyskyt tech konstant. Ale nejsem si jisty, zda v te anotaci jdou konstanty pouzit. Muzes to prosim zkusit?

Reply:
jdou pouzit :) diky